scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -205,81 +139,38 @@ _deploy_deb()
     eval "$cmd"
 }
 
-_install_user_mode_centos()
-{
-    if [[ "$INSTALL_DRIVER_VERSION" == "21.29.20389" ]]; then
-        CMDS=("dnf install --refresh -y intel-opencl-21.29.20389-i593.el8.x86_64 \
-               intel-media-21.2.2-i593.el8.x86_64 \
-               level-zero-1.4.1-i593.el8.x86_64 \
-               intel-level-zero-gpu-1.1.20389-i593.el8.x86_64 \
-               intel-igc-opencl-1.0.7862-i593.el8.x86_64 \
-               intel-igc-core-1.0.7862-i593.el8.x86_64 \
-               intel-ocloc-21.29.20389-i593.el8.x86_64 \
-               intel-gmmlib-21.2.1-i593.el8.x86_64 \
-               ocl-icd-2.2.12-1.el8.x86_64")
-    
-        for cmd in "${CMDS[@]}"; do
-            echo "$cmd"
-            eval "$cmd"
-            if [[ $? -ne 0 ]]; then
-                echo "ERROR: failed to run $cmd" >&2
-                echo "Problem (or disk space)?" >&2
-                echo "                sudo -E $0" >&2
-                echo "Verify that you have enough disk space, and run the script again." >&2
-                exit $EXIT_FAILURE
-            fi
-        done
-    else
-        _deploy_rpm "intel*.rpm"
+_install_user_mode_redhat()
+{   
+    CMDS=("dnf install --refresh -y intel-igc-opencl-1.0.9441-i643.el8.x86_64 \
+           intel-media-21.4.1-i643.el8.x86_64 \
+           level-zero-1.6.2-i643.el8.x86_64 \
+           intel-level-zero-gpu-1.2.21786-i643.el8.x86_64 \
+           intel-opencl-21.49.21786-i643.el8.x86_64 \
+           intel-igc-core-1.0.9441-i643.el8.x86_64 \
+           intel-ocloc-21.49.21786-i643.el8.x86_64 \
+           ocl-icd-2.2.12-1.el8.x86_64 \
+           intel-gmmlib-21.3.3-i643.el8.x86_64")
+
+    for cmd in "${CMDS[@]}"; do
+        echo "$cmd"
+        eval "$cmd"
         if [[ $? -ne 0 ]]; then
-            echo "ERROR: failed to install rpms $cmd error"  >&2
-            echo "Make sure you have enough disk space or fix the problem manually and try again." >&2
+            echo "ERROR: failed to run $cmd" >&2
+            echo "Problem (or disk space)?" >&2
```

Comment:
Can we remove this question? It may be better to call dnf with option --verbose?

Reply:
Let's go through driver update in this review.
This question need further investigation - and can be separately done